### ISSUE TITLE
stdbuf from coreutils

### ISSUE THREAD

**Issue by ustcpetergu** [termux/termux-packages#5245]
Hi, I've been using the `stdbuf` command from the `coreutils` package to de-buffer some commands, but after upgrading `coreutils` to 8.32, it no longer contains `stdbuf`: it's not in built-in programs shown in `coreutils --help`. Is this a bug? 
In the default Termux installation(com.termux 0.92 downloaded from F-Droid) `stdbuf` exists and works fine.

**Comment by xeffyr** [comment]
It is automatically being disabled at package build time:
```
...
checking whether this system supports stdbuf... no
...
```

**Comment by xeffyr** [comment]
This happens due to failure in configuration script caused by https://github.com/termux/termux-packages/blob/master/packages/coreutils/build.sh#L27, so actually can be considered as bug.
```
config.log
...
In file included from <built-in>:366:
<command line>:1:25: warning: missing terminating '"' character [-Winvalid-pp-token]
#define DEFAULT_TMPDIR \"/data/data/com.termux/files/usr/tmp\"
                        ^
1 warning generated.
configure:73592: $? = 0
configure: failed program was:
```

**Comment by xeffyr** [comment]
`stdbuf` is available in 8.32-1.
`pkg upgrade`.

**Comment by ustcpetergu** [comment]
Now it works fine! Thanks a lot for your fast reply!
